script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -1157,7 +1182,7 @@ I don't know whether it will have any effect on you, but I'll try to give you so
 [sha:6044fea21b4aaf93c5182d33398d3077e8761d46]{
 -- Page 133, Offset 8903.
 -- はい、と数枚の一万円札を差し出すアルクェイド。
-'Here', says Arcueid as she hands over a few 10,000 yen bills.
+Arcueid presents me with a handful of 10,000 yen bills.
```

Comment:
This right here is why Arcueid is the best GF in this game

Reply:
Cosmic inflation, monetary inflation... she does it all